doc/source/user_guide/timeseries.rst

```diff
@@ -1563,19 +1563,16 @@ end of the interval is closed:
 
    ts.resample('5Min', closed='left').mean()
 
-Parameters like ``label`` and ``loffset`` are used to manipulate the resulting
-labels. ``label`` specifies whether the result is labeled with the beginning or
-the end of the interval. ``loffset`` performs a time adjustment on the output
-labels.
+Parameters like ``label`` are used to manipulate the resulting labels.
```

Comment:
can you add a new section showing how to use origin (pretty much copy your whatsnew example and put it in the appropriate location). the idea is to have more full examples in the main docs and have a really simple example in whatsnew and point to the main docs.

Reply:
Yep, this is done in the last commit. "What's New" and "Timeseries" is up to date. :wink: